lib/styles/models/box.less

```diff
@@ -102,7 +100,6 @@
     }
     .@{prefix}-content {
         overflow: hidden;
-        display: inline-block;
```

Comment:
same as above here. might be there for reset purposes.

Reply:
can you try something, does it give the same result, if you remove this (display: inline-block) and keep the overflows above?

Reply:
I found that removing either `overflow` or `display` property will fix the scroll problem.
It seemed appropriate for me to remove the overflow property because that is something that directly affects the scrollbar.
I have removed the `display` property as well because it was not causing any effect in display. 
I recommend to remove the `overflow` while we can keep the `display` property.

Reply:
ok that's the spirit, we should remove as less as possible. :) we may add though. especially with some sites, we will need to reset certain items to their defaults over and over.

Reply:
I'd say remove display inline-block; and see if it changes anything in the graphjs.com/docs

Reply:
@esokullu I have tried that as well and checked all the available widgets, and I do not see any negative impact on any widget.